files/bootstrap.sh

```diff
@@ -498,6 +504,16 @@ else
     exit 1
 fi
 
+if [[ -n "${KUBELET_EXTRA_JSON_CONFIG}" ]]; then
+    TMP_KUBE_CONF='/tmp/kubelet-conf-temp.json'
+    KUBELET_CONFIG_EXTRA='/tmp/kubelet-conf-extra.json'
+    echo '%s' > ${KUBELET_CONFIG_EXTRA}
```

Comment:
If we take in a pointer to a file, could we change this entire section to something simpler like - 
```
if [[ -n "${KUBELET_EXTRA_JSON_CONFIG}" ]]; then
    echo "bootstrap.sh: merging user json options into kubelet-config.json"
    jq -s '.[0] + .[1]' "${KUBELET_CONFIG}" "${KUBELET_EXTRA_JSON_CONFIG}" > $KUBELET_CONFIG
fi
```

`jq` doesn't produce a partial result, so we shouldn't need to redirect to a temp file and then copy it over unless I'm missing something else :)

Reply:
Yeah if I remember correctly, this indirection was added because of a different thing. But it was a couple months ago now, so I'll try to remember why.